airmozilla/main/templates/main/event.html

```diff
@@ -15,15 +15,15 @@
 {% endblock %}
 
 {% block extra_head %}
+{{ super() }}
 <meta name="keywords" content="{{ ','.join(tags) }}">
 <meta property="og:title" content="{{ event.title }}">
 <meta property="og:url" content="{{ make_absolute(url('main:event', event.slug)) }}">
 <meta property="og:description" content="{{ short_desc(event, strip_html=True) }}">
-{# these are the same dimensions as used in Roku thumbnails #}
 {% if event.picture %}
-  {% set thumb = thumbnail(event.picture.file, '385x218', crop='center') %}
+  {% set thumb = thumbnail(event.picture.file, '200x200', crop='center') %}
 {% elif event.placeholder_img %}
-  {% set thumb = thumbnail(event.placeholder_img, '385x218', crop='center') %}
+  {% set thumb = thumbnail(event.placeholder_img, '200x200', crop='center') %}
 {% else %}
```

Comment:
You might have done a merge conflict incorrectly. See the diff for the 3 lines you accidentally reverted.